src/site-switcher/site-switcher.html

```diff
@@ -0,0 +1,13 @@
+<div class="dropdown miq-siteswitcher">
+  <button class="btn btn-link dropdown-toggle" type="button" id="domain-switcher" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+    <span class="fa fa-lg fa-th miq-siteswitcher-icon"></span>
+  </button>
+  <div class="dropdown-menu dropdown-menu-right" aria-labelledby="domain-switcher">
+    <div ng-repeat="site in ctrl.sites" class="miq-siteswitcher-entry">
+      <a class="miq-siteswitcher-link" href="{{site.url}}" target="_blank">
```

Comment:
Use `ng-href` instead of `href`, there might be a problem that URLs will be fetched asynchronously and user will click on such URL and it will lead him to 404.
```html
<a class="miq-siteswitcher-link" ng-href="{{site.url}}" target="_blank">
```